common/rfb/CSecurityTLS.cxx

```diff
@@ -238,7 +238,10 @@ void CSecurityTLS::setParam()
     const char *err;
 
 #if GNUTLS_VERSION_NUMBER >= 0x030603
-    ret = gnutls_set_default_priority_append(session, kx_anon_priority, &err, 0);
+    // gnutls_set_default_priority_appends() expects a normal priority string that
+    // doesn't start with ":".
+    static const char kx_anon_priority_normal[] = "+ANON-ECDH:+ANON-DH";
```

Comment:
I'll leave it up to you if you want to add or remove the `:`, but I'd like to avoid duplication if we can. So please try to use the same base string and modify things as needed.